tests/integration/conftest.py

```diff
@@ -0,0 +1,21 @@
+import pytest
```

Comment:
This file necessary to configure django-webtest's pytest integration because we don't use the django auth middleware